src/util/screening-selector.js

```diff
@@ -24,8 +29,14 @@ export async function getPatientCarePlan(client, patientId) {
 
 export async function getQuestionnaireResponsesForPatient(client, patientId) {
   if (!client || !patientId) return null;
+  // we need fresh data
   const questionnaireResponsesResult = await client
-    .request(`QuestionnaireResponse?patient=${patientId}`)
+    .request({
+      url: `QuestionnaireResponse?patient=${patientId}`,
+      headers: {
```

Comment:
@ivan-c or @pbugni does our HAPI Fhir server accepts a request containing a `Cache-Control` header? If so, is it correctly set here? I am adding it because I want to make sure that CQL logic library is operating on a fresh set of questionnaire responses, not cached ones.